src/MCPClient/lib/settings/common.py

```diff
@@ -16,32 +16,186 @@
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
 import ConfigParser
+import StringIO
+
+from django.core.exceptions import ImproperlyConfigured
+
+from env_configparser import EnvConfigParser
+
+
+class Config(object):
```

Comment:
This is the configuration class I've used in MCPClient, MCPServer and Dashboard with the purpose of keeping backward compatibility with the old configuration files while we add support for environment variables.